ts/components/session/AccentText.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import { LocalizerType } from '../../types/Util';
+import { SessionHtmlRenderer } from './SessionHTMLRenderer';
+
+interface Props {
+  i18n: LocalizerType;
```

Comment:
`i18n` is used as global in quite a few places, so maybe we could always assume it is available (and add it to `global.d.ts`)?

Reply:
Updated it in all our typescript files under ts/session/